model/project_parser.go

```diff
@@ -902,18 +904,18 @@ func retrieveFileForModule(ctx context.Context, opts GetProjectOpts, modules Mod
 	}
 
 	// If provided a patch to repeat, use the same githash as the original patch did for the module.
```

Comment:
Is this comment still accurate in that this only applies to repeated patches? It looks like using the base version manifest fixes a bug with how patches locate module revisions in general, repeated or not. Can we clarify/update it?

Reply:
Updated!